tests/keras/layers/test_advanced_activations.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from numpy.testing import assert_allclose
+import numpy as np
+from keras import backend as K
+
+np.random.seed(1337)
+
+def get_standard_values():
+    '''
+    These are just a set of floats used for testing the activation
+    functions, and are useful in multiple tests.
+
+    The values should all be non-negative becasue they and their negative
+    are used to test the ReLU derivates in this file.
+    '''
+    return np.array([[0, 0.1, 0.5, 0.9, 1.0, 10, 1e3, 0.0001]], dtype=K.floatx())
+
+class TestAdvancedActivations(unittest.TestCase):
+    def test_leaky_relu(self):
+        from keras.layers.advanced_activations import LeakyReLU
+        inp = get_standard_values()
+        for alpha in [0., .5, -1.]:
+            layer = LeakyReLU(alpha=alpha)
+            layer.input = K.variable(inp)
+            for train in [True, False]:
+                outp = K.eval(layer.get_output(train))
+                assert_allclose(outp,inp)
+
+            layer.input = K.variable(-inp)
+            for train in [True, False]:
+                outp = K.eval(layer.get_output(train))
+                assert_allclose(outp,-inp*alpha)
+
+            config = layer.get_config()
+            assert config['alpha'] == alpha
+
+    def test_prelu(self):
+        from keras.layers.advanced_activations import PReLU
```

Comment:
imports inside functions are considered a bad practice:
http://stackoverflow.com/questions/1188640/good-or-bad-practice-in-python-import-in-the-middle-of-a-file
Particular issues I got into - there is a code that happens on file import and you get into the mess on who is executing it and when. 

Sometimes they are unavoidable (e.g. keras backends) though. However here it doesn't seem to be the case